spyder/plugins/__init__.py

```diff
@@ -37,6 +37,7 @@
 from spyder.py3compat import configparser, is_text_string
 from spyder.utils import icon_manager as ima
 from spyder.utils.qthelpers import create_action, toggle_actions
+from spyder.plugins.configdialog import PluginConfigPage  # Plugin compatibility

```

Comment:
I mean, the thing is when you do

    from spyder.plugins import configdialog

that causes `configdialog` to import itself again from here.